tests/i18n/test_extraction.py

```diff
@@ -534,27 +534,25 @@ def tearDown(self):
         os.chdir(self._cwd)
 
     def test_symlink(self):
-        # On Python < 3.2 os.symlink() exists only on Unix
-        if hasattr(os, 'symlink'):
-            if os.path.exists(self.symlinked_dir):
-                self.assertTrue(os.path.islink(self.symlinked_dir))
-            else:
-                # On Python >= 3.2) os.symlink() exists always but then can
-                # fail at runtime when user hasn't the needed permissions on
-                # Windows versions that support symbolink links (>= 6/Vista).
-                # See Python issue 9333 (http://bugs.python.org/issue9333).
-                # Skip the test in that case
-                try:
-                    os.symlink(os.path.join(self.test_dir, 'templates'), self.symlinked_dir)
-                except (OSError, NotImplementedError):
-                    raise SkipTest("os.symlink() is available on this OS but can't be used by this user.")
-            os.chdir(self.test_dir)
-            management.call_command('makemessages', locale=[LOCALE], verbosity=0, symlinks=True)
-            self.assertTrue(os.path.exists(self.PO_FILE))
-            with open(self.PO_FILE, 'r') as fp:
-                po_contents = force_text(fp.read())
-                self.assertMsgId('This literal should be included.', po_contents)
-                self.assertIn('templates_symlinked/test.html', po_contents)
+        if os.path.exists(self.symlinked_dir):
+            self.assertTrue(os.path.islink(self.symlinked_dir))
+        else:
+            # On Python >= 3.2) os.symlink() exists always but then can
+            # fail at runtime when user hasn't the needed permissions on
+            # Windows versions that support symbolink links (>= 6/Vista).
+            # See Python issue 9333 (http://bugs.python.org/issue9333).
+            # Skip the test in that case
+            try:
+                os.symlink(os.path.join(self.test_dir, 'templates'), self.symlinked_dir)
```

Comment:
My bad. I forgot completely about Python 2 and broke it on Windows. I've undone that extra bogus change on a new commit.